deployment/modules/gcp/example-gcp/main.tf

```diff
@@ -0,0 +1,134 @@
+terraform {
+  backend "gcs" {}
+}
+
+## Call the Tessera GCP module
+##
+## This will configure all the storage infrastructure required to run a Tessera log on GCP.
+module "gcs" {
+  source = "..//gcs"
```

Comment:
Is the double slash needed here? I think this is one of those cases where there is something needed about the 2 slashes but best to check.

Reply:
yup, same as above.